### ISSUE TITLE
[BUG] Specified Search Engine was not Found: 'custom'

### ISSUE THREAD

**Issue by katacarbix** [Lissy93/dashy#267]
### Environment

Self-Hosted (Docker)

### Version

1.8.5

### Describe the problem

I am trying to add web searching to my dashboard. I want it to use my personal Searx instance but Dashy seems to be having trouble with the `custom` search engine type. When I type my search query it says "No search results" (I think that's just for searching the items on my dashboard) but when I press Enter nothing seems to happen but in the web console it gives me the error message in the screenshot below. I get the same exact behavior in Chromium as I do in Firefox.

#### Dashy webSearch configuration:
![Nested JSON objects with `searchEngine: custom` and other fields from the webSearch docs (https://github.com/Lissy93/dashy/blob/master/docs/configuring.md#appconfigwebsearch-optional)](https://user-images.githubusercontent.com/1340784/135948322-10e11c9d-8c7c-464f-b1bc-63f8b3184fa6.png)

#### Firefox console message:
![⚠️ Warning ⚠️ Specified Search Engine was not Found: 'custom' This is likely not an issue with Dashy, but rather your configuration. If you think it is a bug, please open a ticket on GitHub: https://git.io/JukXk](https://user-images.githubusercontent.com/1340784/135948087-9a5ca73c-a4bf-477d-a81b-146dde17c4c1.png)

### Please tick the boxes

- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy (check the first two digits of the version number)
- [X] You've checked that this [issue hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by Lissy93** [comment]
Yup, this is a bug - thank you for raising. I will try and push a fix either this evening or tommorrow evening :)

In case anyone else wants to pick this up, see `searchSubmitted(){...}` of [`SearchBar.vue`](https://github.com/Lissy93/dashy/blob/master/src/components/Settings/SearchBar.vue#L122). Line 132 should be updated to something like this:

```javascript
const isCustomSearch = (searchPrefs.searchEngine === 'custom' && searchPrefs.customSearchEngine);

let searchUrl = isCustomSearch ?
    searchPrefs.customSearchEngine
    : findUrlForSearchEngine(desiredSearchEngine, searchEngineUrls)
```

**Comment by Lissy93** [comment]
Hey @katacarbix this is now fixed in #269 :)

**Comment by katacarbix** [comment]
Awesome, thank you so much!!! 🎉
